support/CMakeLists.txt

```diff
@@ -0,0 +1,77 @@
+# TODO
+# CONFIG += ensure_skins 
+# CONFIG += fontconfig
+# CONFIG += qskinny
+
+list(APPEND TARGET_HEADERS
+    SkinnyGlobal.h
+    SkinnyNamespace.h
+    SkinnyShapeFactory.h
+    SkinnyShapeProvider.h
+    SkinnyShortcut.h)
+
+list(APPEND TARGET_SOURCES
+    SkinnyNamespace.cpp
+    SkinnyShapeFactory.cpp
+    SkinnyShapeProvider.cpp
+    SkinnyShortcut.cpp)
+
+# TODO
+#fontconfig{
+#
+#    QSK_FONTDIR = $${PWD} / fonts QSK_FONTCACHEDIR =
+#                      $${OUT_PWD} / fontconfig QSK_FONTCONF_FILE =
+#                          $${QSK_FONTCACHEDIR} / fonts.conf
+#
+#                                                       QSK_FONTCONF_DATA =
+#                              "$$cat($${PWD}/fonts.conf.in, false)" QSK_FONTCONF_DATA = $$replace(
+#                                  QSK_FONTCONF_DATA, FONTCACHEDIR, $${QSK_FONTCACHEDIR})
+#                                  QSK_FONTCONF_DATA = $$replace(
+#                                      QSK_FONTCONF_DATA, FONTDIR, $${QSK_FONTDIR})
+#                                      write_file($${QSK_FONTCONF_FILE}, QSK_FONTCONF_DATA)
+#
+#                                          DEFINES +=
+#    FONTCONFIG_FILE = $$clean_path($$QSK_FONTCONF_FILE)
+#}
+
+set(TARGET_NAME qsktestsupport)
```

Comment:
@uwerat is this library required in order to use QSkinny or is it more like a comfort layer?

Reply:
It is needed for the examples - but not intended for users.